src/ImportDefinitionsBundle/Interpreter/CarbonInterpreter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Import Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2016-2018 w-vision AG (https://www.w-vision.ch)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+namespace ImportDefinitionsBundle\Interpreter;
+
+use ImportDefinitionsBundle\Model\DataSetAwareInterface;
+use ImportDefinitionsBundle\Model\DataSetAwareTrait;
+use ImportDefinitionsBundle\Model\DefinitionInterface;
+use ImportDefinitionsBundle\Model\Mapping;
+use Pimcore\Model\DataObject\Concrete;
+
+class CarbonInterpreter implements InterpreterInterface, DataSetAwareInterface
+{
+    use DataSetAwareTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function interpret(Concrete $object, $value, Mapping $map, $data, DefinitionInterface $definition, $params, $configuration)
+    {
+        if ($value) {
+            return new \Carbon\Carbon($value);
```

Comment:
Qualifier is unnecessary and can be replaced with an import.

```php
use Carbon\Carbon;
...
return new Carbon($value);
```

Reply:
I changed it. I have oriented to QuantityValueInterpreter :)